lib/geordi/util.rb

```diff
@@ -62,21 +62,29 @@ def system!(*commands)
         end
       end
 
+      def binstub(executable, arguments='')
+        binstub_file = "bin/#{executable}"
+
+        command = 'bundle exec '
+        command << File.exists?(binstub_file) ? binstub_file : executable
+        command << ' ' + arguments
+      end
+
       def console_command(environment)
         if gem_major_version('rails') == 2
           'script/console ' + environment
         elsif gem_major_version('rails') == 3
-          'bundle exec rails console ' + environment
+          binstub 'rails', "console #{environment}"
```

Comment:
If you define the binstub method as `def binstub(executable, *args)`, it can be used flexibly here like `binstub 'rails', 'console', environment`. wdyt?